elyra/metadata/schemas/kfp.json

```
@@ -67,6 +68,15 @@
             "category": "Kubeflow Pipelines"
           }
         },
+        "engine": {
+          "title": "Kubeflow Pipelines engine",
+          "description": "The Kubeflow Pipelines engine in use (Argo or Tekton)",
+          "type": "string",
```

We should leverage schema constructs for this.  With the following, if `engine` does not specify a value, a value of `"Argo"` will be used.  It will also automatically enforce only `"Argo"` and `"Tekton"` can be set as values.
```suggestion
          "type": "string",
          "enum": ["Argo", "Tekton"],
          "default": "Argo",
```